spell/rwa-onboarding-checklist.md

```diff
@@ -11,64 +12,73 @@
     * [ ] `RwaUrn2`
       * [ ] contract is verified on etherscan
         * [ ] ensure `0.6.12` solc version is used
-        * [ ] ensure optimization are off
-        * [ ] ensure license is specified
-        * [ ] ensure source matches github code and it's consistent with previous RWA onboarding
-      * [ ] constructor args are correct and match [chainlog](https://chainlog.makerdao.com/)
-        * [ ] `vat`
-        * [ ] `jug`
-        * [ ] `gemJoin`
-        * [ ] `daiJoin`
-        * [ ] `outputConduit` matches `RwaOutputConduit<2,3>`
+        * [ ] ensure optimizations are off
+        * [ ] ensure license is specified (SPDX in code or otherwise)
+        * [ ] ensure source matches [GitHub code](https://github.com/makerdao/rwa-toolkit/blob/92c79aac24ef7645902ce4be57ba41b19e6c7dd5/src/urns/RwaUrn2.sol) and it's consistent with previous RWA onboardings
+      * [ ] constructor args are correct and match [Chainlog](https://chainlog.makerdao.com/)
+        * [ ] `vat` matches `MCD_VAT`
+        * [ ] `jug` matches `MCD_JUG`
+        * [ ] `gemJoin` matches `MCD_JOIN_RWAXXX_A` (Per spell code)
+        * [ ] `daiJoin` matches `MCD_JOIN_DAI`
+        * [ ] `outputConduit` matches `RwaOutputConduit<2,3>` (Per spell code)
       * [ ] check `wards`
         * [ ] `MCD_PAUSE_PROXY` is relied
+        * [ ] `MCD_ESM` is being relied in the spell (as approved by GovAlpha)
         * [ ] deployer is denied
         * [ ] no other address has been relied
     * [ ] `RwaJar`
       * [ ] contract is verified on etherscan
         * [ ] ensure `0.6.12` solc version is used
         * [ ] ensure optimization are off
-        * [ ] ensure license is specified
-        * [ ] ensure source matches github code and it's consistent with previous RWA onboarding
-      * [ ] constructor arg matches [chainlog](https://chainlog.makerdao.com/)
+        * [ ] ensure license is specified (SPDX in code or otherwise)
+        * [ ] ensure source matches [GitHub code](https://github.com/makerdao/rwa-toolkit/blob/92c79aac24ef7645902ce4be57ba41b19e6c7dd5/src/jars/RwaJar.sol) and it's consistent with previous RWA onboarding
+      * [ ] constructor arg matches [`ChainLog`](https://chainlog.makerdao.com/)
         * [ ] `chainlog`
       * [ ] no `wards`
     * [ ] `RwaInputConduit<2,3>`
       * [ ] contract is verified on etherscan
         * [ ] ensure `0.6.12` solc version is used
         * [ ] ensure optimization are off
-        * [ ] ensure license is specified
-        * [ ] ensure source matches github code and it's consistent with previous RWA onboarding
+        * [ ] ensure license is specified (SPDX in code or otherwise)
+        * [ ] ensure source matches [GitHub code](https://github.com/makerdao/rwa-toolkit/blob/92c79aac24ef7645902ce4be57ba41b19e6c7dd5/src/conduits/RwaSwapInputConduit2.sol) and it's consistent with previous RWA onboarding
       * [ ] constructor args are correct and match [chainlog](https://chainlog.makerdao.com/)
```

Comment:
Should we add `quitTo` addresses here? Or are they too context dependent?

Reply:
`quitTo` is meant to be used as an escape by the operator in case the PSM is full. If that happens, they can withdraw the stablecoins to the `quitTo` address using `quit()`.
Usually this address is the same as the `to` address for the output conduit, but that might not hold true every single time.

Worst case scenario, governance can always `yank` any gem in the input conduit, so in theory it's fine to not have this set.

Also notice that `quitTo` is `file`able, but it's not a constructor argument.